client/jetpack-onboarding/steps/business-address.jsx

```diff
@@ -50,11 +50,11 @@ class JetpackOnboardingBusinessAddressStep extends React.PureComponent {
 		const { translate } = this.props;
 
 		return {
-			name: translate( 'Business Name' ),
-			street: translate( 'Street Address' ),
+			name: translate( 'Business name' ),
+			street: translate( 'Street address' ),
 			city: translate( 'City' ),
-			state: translate( 'State' ),
-			zip: translate( 'ZIP Code' ),
+			state: translate( 'State / Region / Province' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **58** times:
`translate( 'State/Province' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).